### PR TITLE
Issue 168

### PR DESCRIPTION
**Pull request by crtrott**

### CODE REVIEW

**Comment by crtrott** [comment]
I will just merge this now. It fixes the OpenMP unit test failures on White in integration testing which were the only "new" failures:

```
    Start 1: MueLu_UnitTestsEpetra_kokkos_MPI_1
1/4 Test #1: MueLu_UnitTestsEpetra_kokkos_MPI_1 ...   Passed    1.39 sec
    Start 2: MueLu_UnitTestsEpetra_kokkos_MPI_4
2/4 Test #2: MueLu_UnitTestsEpetra_kokkos_MPI_4 ...   Passed    2.38 sec
    Start 3: MueLu_UnitTestsTpetra_kokkos_MPI_1
3/4 Test #3: MueLu_UnitTestsTpetra_kokkos_MPI_1 ...   Passed    1.30 sec
    Start 4: MueLu_UnitTestsTpetra_kokkos_MPI_4
4/4 Test #4: MueLu_UnitTestsTpetra_kokkos_MPI_4 ...   Passed    9.33 sec

100% tests passed, 0 tests failed out of 4

Label Time Summary:
MueLu    =  14.39 sec (4 tests)

Total Test time (real) =  14.43 sec
```

File: src/graph/KokkosGraph_graph_color.hpp [hunk 1]
```diff
@@ -211,7 +211,14 @@ void d2_graph_color(
     }
 
     default:
+    {
+      color_view_type colors_out = color_view_type("Graph Colors", num_rows);
```
Comment (ndellingwood) [inline src/graph/KokkosGraph_graph_color.hpp]: @crtrott Just to check, Serial here refers strictly to the algorithms style of execution, the exec_space of the View does not need to be Kokkos::Serial?

Comment (crtrott) [reply src/graph/KokkosGraph_graph_color.hpp]: Yeah exactly.

**Comment by ndellingwood** [comment]
Cross-reference issue #168

**Pull request merged**